dae/dae/genomic_resources/tests/test_score_statistics_state.py

```diff
@@ -54,105 +53,105 @@ def proto_fixture(tmp_path):
     return build_filesystem_test_protocol(tmp_path)
 
 
-def test_double_build_histograms_without_change(proto_fixture, dask_client):
+# def test_double_build_histograms_without_change(proto_fixture, dask_client):
```

Comment:
If that test doesn't work anymore either fix it or remove it.

Reply:
Deleted.